misc/testerina/modules/testerina-core/src/main/ballerina/src/test/Module.md

```diff
@@ -114,6 +114,25 @@ function testMockImportedFunction() {
 }
 ```
 
+Scoping and stubbing of mock functions declared for import functions is also supported
+
+```ballerina
+import ballerina/math;
+import ballerina/test;
+
+@test:Mock {
+    moduleName : "ballerina/math",
+    functionName : "absInt"
+}
+test:MockFunction mock_absInt = new();
+
+@test:Config {}
+public function testFunction() {
+    test:when(mock_absInt).thenReturn(100);
+    test:assertEquals(math:absInt(-5), 100);
+}
+```
+
 #### Mocking a function in the same module
```

Comment:
Since the feature is generalized to handle both imported functions and functions in the same module using the `test:MockFunction`, we don't need this separation now. We can put a general description under the title `### Function Mocking`

Reply:
Changed it so that it we use a general description for both